test/unit/application_record_test.rb

```diff
@@ -0,0 +1,23 @@
+require "test_helper"
+
+class ApplicationRecordTest < ActiveSupport::TestCase
+  test "append_url_options adds locale" do
+    assert_equal "/government/foo.cy", Edition.new.append_url_options("/government/foo", locale: "cy")
```

Comment:
at what point will we be able to remove this file? Just thinking out loud given we're adding functionality to append locales

Reply:
TBH, I think this will probably stay here.  It's a method that's inherited by nearly all models, so the `ApplicationRecord` seemed the best place to include this.